storage/src/factory.rs

```diff
@@ -117,6 +117,10 @@ impl BlobFactory {
     ) -> IOResult<Arc<dyn BlobCache>> {
         let backend_cfg = config.get_backend_config()?;
         let cache_cfg = config.get_cache_config()?;
+        let user_io_batch_size = config
+            .get_rafs_config()
+            .map_or_else(|_| default_user_io_batch_size(), |v| v.user_io_batch_size)
```

Comment:
Should be?
```
.map_or_else(|| default_user_io_batch_size(), |v| v.user_io_batch_size)
```

Reply:
function is expected to take 1 argument, but it takes 0 arguments
expected function that takes 1 argument